src/content/docs/reference/core.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Membrane core library
+---
+
+Programs written in Membrane are mostly just vanilla TypeScript/JavaScript, but there are a handful of patterns to get familiar with.
+
+## Action
+
+[Actions](/concepts/schema#actions) exported from Membrane programs can be invoked manually in the IDE Navigator or programmatically with the core lib.
+
+:::tip
+You can also simply `await` an action to invoke it, which is shorthand for chaining `.$invoke()` and awaiting.
+E.g. `await nodes.sms.send({ message: "Bienvenidos a Membrane" })`
+:::
+
+````ts
+interface Action<V> {
+  /**
+   * Invoke this action
+   */
+  $invoke(): Promise<V>;
+  /**
+   * Schedules a timer to invoke the action at a specific time.
+   * @param time The time to invoke the action, either a UNIX timestamp or a Date object.
+   */
+  $invokeAt(time: number | Date): void;
+  /**
+   * Schedules a timer to invoke the action after a specified delay.
+   * @param seconds The number of seconds to wait before invoking the action.
+   */
+  $invokeIn(seconds: number): void;
+  /**
+   * Schedules a cron-like timer that invokes the action at the specified times.
+   *
+   * @param spec A cron-like expression that determines when this timer fires. Unlike traditional cron, this parameter has
+   * seconds-level accuracy, so a cron expression must specify 6 values (instead of 5):
+   *
+   * Examples:
+   *  - `* * * * * *`   - Every second
+   *  - `0 * * * * *`   - Every minute (at the :00 second mark)
+   *  - `0 3 * * * 4`   - At 3:00 AM every Thursday (4 = Thursday)
+   *  - `0 45 19 * Mar *` - Every year at 7:45 PM on the first of March
+   *
+   * Valid values for each field are:
+   * ```
+   * second        0-59
+   * minute        0-59
+   * hour          0-23
+   * day of month  1-31
+   * month         1-12 (or names: Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct, Nov, Dec)
+   * day of week   0-7 (both 0 or 7 represent Sunday, or names: Sun, Mon, Tue, Wed, Thu, Fri, Sat)
+   * ```
+   */
+  $cron(spec: string): void;
+}
+````
```

Comment:
I think we should break each member into their own subsections. Not only the tsdoc comments are not super readable but by breaking it down, we'd be able to link to specific methods directly.

I'm picturing this sidebar
![Screenshot 2024-08-09 at 11 38 33](https://github.com/user-attachments/assets/1236530d-109d-4148-ad3c-b27d0200927e)

Showing a tree like:

```
Overview
Exports
  state
  root
  nodes
Types
  Action
    $invoke
    $invokeIn
    $invokeAt
    $cron
  Field
    ...
```